protocol/src/main/java/org/fido/iot/protocol/CryptoService.java

```diff
@@ -531,9 +533,16 @@ public PublicKey decode(Composite pub) {
    *
    * @param verificationKey The verification key to use.
    * @param cose            The COSE message.
+   * @param sigInfoA        The sigInfo object representing eA
    * @return True if the signature matches otherwise false.
    */
-  public boolean verify(PublicKey verificationKey, Composite cose) {
+  public boolean verify(PublicKey verificationKey, Composite cose, Composite sigInfoA) {
+    if (null == verificationKey && null != sigInfoA
+        && Arrays.asList(Const.SG_EPIDv10, Const.SG_EPIDv11, Const.SG_EPIDv20)
+        .contains(sigInfoA.getAsNumber(Const.FIRST_KEY))) {
```

Comment:
Can you use Arrays.contains directly instead of first converting to a list?

Reply:
There is no Arryas.contains. We could use stream though, but this seems much simpler and readable.